onchain/rollups/contracts/consensus/quorum/Quorum.sol

```diff
@@ -0,0 +1,152 @@
+// (c) Cartesi and individual authors (see AUTHORS)
+// SPDX-License-Identifier: Apache-2.0 (see LICENSE)
+
+pragma solidity ^0.8.8;
+
+import {BitMaps} from "@openzeppelin/contracts/utils/structs/BitMaps.sol";
+
+import {AbstractConsensus} from "../AbstractConsensus.sol";
+import {InputRange} from "../../common/InputRange.sol";
+
+/// @notice A consensus model controlled by a small, immutable set of `n` validators.
+/// @notice You can know the value of `n` by calling the `numOfValidators` function.
+/// @notice Upon construction, each validator is assigned a unique number from 1 and `n`.
```

Comment:
`from 1 to n` or `between 1 and n`

Reply:
Maybe `between 1 and n`, right?

Reply:
I actualy like `from 1 to n` because its how they are ordered and assigned.` between 1 and n` doesn't convey order and its IMO.